scripts/start.sh

```diff
@@ -13,308 +16,251 @@ TZ=${TZ:-UTC}
 
-if  [ ! -d /data/certs ]; then
+if [ ! -d /var/lib/gvm/CA ] || [ ! -d /var/lib/gvm/private ] || [ ! -d /var/lib/gvm/private/CA ] ||
+	[ ! -f /var/lib/gvm/CA/cacert.pem ] || [ ! -f /var/lib/gvm/CA/clientcert.pem ] ||
+	[ ! -f /var/lib/gvm/CA/clientcert.pem ] || [ ! -f /var/lib/gvm/private/CA/cacert.pem ] ||
+	[ ! -f /var/lib/gvm/private/CA/clientcert.pem ] || [ ! -f /var/lib/gvm/private/CA/clientcert.pem ]; then
 	echo "Creating certs folder..."
-	mkdir -p /data/certs/CA
-	mkdir -p /data/certs/private
-	
+	mkdir -p /var/lib/gvm/CA
+	mkdir -p /var/lib/gvm/private
+
 	echo "Generating certs..."
 	gvm-manage-certs -a
-	
-	cp /usr/local/var/lib/gvm/CA/* /data/certs/CA/
-	
-	cp -r /usr/local/var/lib/gvm/private/* /data/certs/private/
-	
-	chown gvm:gvm -R /data/certs
-fi
 
-if [ ! -h /usr/local/var/lib/gvm/CA ]; then
-	echo "Fixing certs CA folder..."
-	rm -rf /usr/local/var/lib/gvm/CA
-	ln -s /data/certs/CA /usr/local/var/lib/gvm/CA
-	
-	chown gvm:gvm -R /data/certs
-	chown gvm:gvm -R /usr/local/var/lib/gvm/CA
+	chown gvm:gvm -R /var/lib/gvm/
 fi
 
-if [ ! -h /usr/local/var/lib/gvm/private ]; then
-	echo "Fixing certs private folder..."
-	rm -rf /usr/local/var/lib/gvm/private
-	ln -s /data/certs/private /usr/local/var/lib/gvm/private
-	chown gvm:gvm -R /data/certs
-	chown gvm:gvm -R /usr/local/var/lib/gvm/private
-fi
-
-if  [ ! -d /data/plugins ]; then
-	echo "Creating NVT Plugins folder..."
-	mkdir /data/plugins
-fi
-
-if [ ! -h /usr/local/var/lib/openvas/plugins ]; then
-	echo "Fixing NVT Plugins folder..."
-	rm -rf /usr/local/var/lib/openvas/plugins
-	ln -s /data/plugins /usr/local/var/lib/openvas/plugins
-	chown gvm:gvm -R /data/plugins
-	chown gvm:gvm -R /usr/local/var/lib/openvas/plugins
-fi
-
-if  [ ! -d /data/cert-data ]; then
-	echo "Creating CERT Feed folder..."
-	mkdir /data/cert-data
-fi
-
-if [ ! -h /usr/local/var/lib/gvm/cert-data ]; then
-	echo "Fixing CERT Feed folder..."
-	rm -rf /usr/local/var/lib/gvm/cert-data
-	ln -s /data/cert-data /usr/local/var/lib/gvm/cert-data
-	chown gvm:gvm -R /data/cert-data
-	chown gvm:gvm -R /usr/local/var/lib/gvm/cert-data
-fi
-
-if  [ ! -d /data/scap-data ]; then
-	echo "Creating SCAP Feed folder..."
-	
-	mkdir /data/scap-data
-fi
-
-if [ ! -h /usr/local/var/lib/gvm/scap-data ]; then
-	echo "Fixing SCAP Feed folder..."
-	
-	rm -rf /usr/local/var/lib/gvm/scap-data
-	
-	ln -s /data/scap-data /usr/local/var/lib/gvm/scap-data
-	
-	chown gvm:gvm -R /data/scap-data
-	chown gvm:gvm -R /usr/local/var/lib/gvm/scap-data
-fi
-
-if  [ ! -d /data/data-objects/gvmd ]; then
-	echo "Creating GVMd Data Objects folder..."
-	
-	mkdir -p /data/data-objects/gvmd
-fi
-
-if [ ! -h /usr/local/var/lib/gvm/data-objects ]; then
-	echo "Fixing GVMd Data Objects folder..."
-	
-	rm -rf /usr/local/var/lib/gvm/data-objects
-	
-	ln -s /data/data-objects /usr/local/var/lib/gvm/data-objects
-	
-	chown gvm:gvm -R /data/data-objects
-	chown gvm:gvm -R /usr/local/var/lib/gvm/data-objects
+if [ ! -f "/var/lib/gvm/.firstsync" ]; then
```

Comment:
This is gated twice, once here and once in the script it's gating.

Either fold the other script in this one, to clearly indicate that this is not something intended to be run by the user, or run it unconditionally here.

Reply:
Thats correct and is a check on stage1 to prevent a multi run on restart etc. on stage2 it should prevent the manual running of the script it self.
So I don't see there any need to change that as it is also in the current image the same.

Reply:
This is part of the "Clean up start.sh" milestone, so this is within the scope of refactoring.

The condition currently shown is
`if [ ! -f "/var/lib/gvm/.firstsync" ]; then`
The condition in the script is
`if [ ! -f "/var/lib/gvm/.firstsync" ] && [ -f "/opt/gvm-sync-data.tar.xz" ]; then`

This is effectively the same as
`if [ ! -f "/var/lib/gvm/.firstsync" ] && [ ! -f "/var/lib/gvm/.firstsync" ] && [ -f "/opt/gvm-sync-data.tar.xz" ]; then`

Which is redundant, the `prevent a multi run on restart etc.` is also accomplished by the second one, using the same condition.